Global input files -- Abdou

trysarahtops opened the pull request:
The README has been updated to include the Abdou data. The scenario folder includes candu_inputs based on the Abdou assumptions for the Global supply, Korean and Canadian candus.

Assumptions from Abdou were coupled with actual historical data to produce the scenario snippets and plot.

The scenario produces the following image:
<img width="1653" height="993" alt="AbdouSupply" src="https://github.com/user-attachments/assets/fa8edd76-b4ea-4bf3-9ef2-687620253b80" />

trysarahtops committed:
Create README.md

Added Abdou scenario to README

trysarahtops committed:
Create AbdouKoreanCandus.xml

input snippet for Korean candus based on Abdou assumptions

trysarahtops committed:
Create AbdouCanadianCandus.xml

input snippet for Canadian candus based on Abdou assumptions

trysarahtops committed:
Create AbdouGlobalT.xml

file to generate global tritium supply based on Abdou assumptions

gonuke reviewed (changes requested):
Thanks for these additions. Some questions about some small differences in structure between these and the Kovari scenario.

Also some QA on what seems like an extra value at the beginning of the deployment times and lifetimes

trysarahtops committed:
updated file to match other scenario format

trysarahtops committed:
Update README.md

Updated information for Abdou recreation assumption for the without fusion scenario.

trysarahtops committed:
Update AbdouGlobalT.xml

Updated Global tritium supply script for Abdou scenario without fusion

trysarahtops committed:
Update AbdouKoreanCandus.xml

Updated Korean CANDU input snippet for Abdou recreation scenario without fusion

trysarahtops committed:
Update AbdouCanadianCandus.xml

Updated Canadian CANDU input snippet for Abdou without fusion scenario

trysarahtops commented:
Updated the input files for this scenario and explained the changes in the markdown file. A comparison between the tricycle simulation and the digitized data is shown below.
<img width="798" height="509" alt="AbdouComparison" src="https://github.com/user-attachments/assets/fabfbf89-07b3-44ff-bfc9-bd136c32014e" />

trysarahtops committed:
Merge branch 'main' into global_input_files_Abdou

trysarahtops committed:
Update fuel cycle flow in AbdouGlobalT.xml

Updated fuel cycle flow to match similar global tritium supply files.

gonuke commented on scenarios/candu_inputs/AbdouKoreanCandus.xml:
```diff
@@ -0,0 +1,89 @@
+    <Source>
+      <outcommod>Tritium</outcommod>
+      <outrecipe>T</outrecipe>
+      <throughput>0.0009508</throughput>
```
This looks like it's got an extra 0?

```suggestion
      <throughput>0.009508</throughput>
```

gonuke reviewed (changes requested):
Just looking for confirmation on the Korean numbers

trysarahtops commented on scenarios/candu_inputs/AbdouKoreanCandus.xml:
```diff
@@ -0,0 +1,89 @@
+    <Source>
+      <outcommod>Tritium</outcommod>
+      <outrecipe>T</outrecipe>
+      <throughput>0.007663</throughput>
```
The values are based on a production factor of 0.2 kg of tritium annually multiplied by the energy availability factor. The design net capacity was used for the power level. For this specific unit, that is 73.1% and only goes up to 2019. The other units have higher energy availability factors (over 80%) and go up to 2024. I can definitely update the README with these values.

Just to double check, this is how I calculated the monthly tritium production for this specific CANDU reactor: $\ 0.2 kg/GW_e/fpy * 0.731 * 0.629 GW / 12 mo = 0. 007663 kg/mo $

gonuke replied:
Understood... so this is based on a low capacity factor/availability of the specific reactor.  Let's discuss this further

trysarahtops committed:
Apply suggestion from @gonuke

Removed extra zero from throughput value

Co-authored-by: Paul Wilson <paul.wilson@wisc.edu>

gonuke commented:
I'm curious to see the impact of fixing the typo in the generation rate for the one Wolsong reactor in the plot comparing to published data

trysarahtops commented:
After the change, this looks a lot better to me.

<img width="640" height="480" alt="Abdou" src="https://github.com/user-attachments/assets/25dc03e9-9f38-463d-858a-9dc78150519f" />

gonuke commented:
Do you know if there is some important assumption about the post-2040 time period that we're missing from the Abdou analysis?  We can probably go forward with this once we resolve the merge conflict, and leave that for a separate review.

trysarahtops committed:
Merge branch 'main' into global_input_files_Abdou
